ros/processor/event_producer.py

```diff
@@ -0,0 +1,76 @@
+import json
+from flask import request
+from ros.lib import produce
+from confluent_kafka import KafkaError
+from datetime import datetime, timezone
+from ros.lib.models import PerformanceProfile
+from ros.lib.config import NOTIFICATIONS_TOPIC, get_logger
+from ros.lib.utils import org_id_from_identity_header, systems_ids_for_existing_profiles
+
+logger = get_logger(__name__)
+
+
+# Event for new suggestion
+def new_suggestion_event(host, platform_metadata, previous_state, current_state):
+
+    org_id = org_id_from_identity_header(request)
+    query = systems_ids_for_existing_profiles(org_id)
+    systems_with_suggestions = query.filter(PerformanceProfile.number_of_recommendations > 0).count()
+    request_id = platform_metadata.get('request_id')
+    payload = {
```

Comment:
Could you extract keys that commonly used for every notification example - `version, bundle, application`?
Then just append those keys which are specific to current event. What do you think?

Reply:
Sorry I didn't get it.